department-of-veterans-affairs/vets-website
# CC-461 combobox keyboard handling

## Summary

- Adds appropriate keyboard handling to the combobox per https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/461

- adds handling for the Tab, Enter, Escape, Spacebar, Up & Down Arrow Keys. 

## Related issue(s)
 https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/461

## Testing done

- Verified that keyboard handling worked as described for each key.

- Enter text into box.
- Up and down arrow keys should change the highlighted element.
- Hit 'Enter'. The highlighted element should be selected, options minimized & focus on the text input.
- Continue typing, the autosuggest should be highlighted. Hit Escape, the options should minimize and updated text should remain the same, focus on the text input.
- Type some more, include a space. The input should take a space no problem. When a different item is selected (via arrow key navigation), space should function the same as Enter (highlighted option selected, options minimized, focus on the text input)
- Type some more, hit `Tab`. The options should minimize, updated text remain the same, `update` or `save` button should have focus.

## What areas of the site does it impact?

526ez form, revised disabilities page

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling
